python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S4423.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "Weak SSL\/TLS protocols should not be used",
+  "type": "VULNERABILITY",
```

Comment:
Shouldn't it be a hotspot?

Reply:
This is a vulnerability as per PM specification, summoning @pierre-loup-tristant-sonarsource for confirmation.

Reply:
It is a vulnerability for Python as it also is for Java and PHP.